src/sql/azdata.proposed.d.ts

```diff
@@ -1338,5 +1342,24 @@ declare module 'azdata' {
 		 * Sets the ariaLabel for the right arrow button that shows up in clickable infoboxes
 		 */
 		clickableButtonAriaLabel?: string;
+
+		/**
+		 * Add links to infobox text.
+		 */
+		links?: LinkArea[];
+	}
+
+	/**
+	 * Event argument for infobox link click event.
+	 */
+	export interface InfoBoxLinkClickEvent {
+		/**
+		 * Index of the link selected
+		 */
+		index: number;
```

Comment:
you have the linkArea below, why do you still need the index?

Reply:
There can be a situation where there are 2 similar links (with same text and url) and the extension handling the event want to differentiate between the 2 links.

For example:
_Click here_ to open dialog 1  or _Click here_ to open dialog 2